testing/cdms2/cdms2_test_createcopy_lose_dtype.py

```diff
@@ -0,0 +1,19 @@
+import cdms2
+import os
+import cdat_info
+
+
+incat = cdms2.open(os.path.join(cdat_info.get_sampledata_path(),"tas_ccsr-95a.xml"))
+invar  = incat['tas'] 
+intype = invar.dtype  # dtype(float32)
+
+outfile = cdms2.open('newfile.nc', 'w')
+outfile.createVariableCopy(invar)
+
+outvar = outfile['tas']
+outtype= outvar.dtype # dtype(float64)
+
+os.remove("newfile.nc")
```

Comment:
@doutriaux1 doesn't this need a `outfile.close()` so that the file handle doesn't prevent deletion?